code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -95,6 +95,14 @@ datum
 					else
 
 						M.TakeDamage("All", 0, min(5, volume * 0.5) * stack_mult, 0, DAMAGE_BURN)
+				else if (method == INGEST)
+					if (ishuman(M))
```

Comment:
these are three sections of identical code, so should probably be a proc on `/mob/living/carbon/human` though it does seem odd that a var defined on mob can only be set on humans

Reply:
As nice as it'd be to deduplicate this, I think the current approach is reasonably functional and makes it easier to tinker with later (i.e. if you want different damage or make one of the acids not capable of scarring) and it'd be weird to have it on the mob side.

Reply:
Think I might leave it as is for now for reasons Kubius mentioned, and also the fact that I'm not sure how to go about making a proc work.